.github/workflows/library.yml

```diff
@@ -347,10 +341,25 @@ jobs:
           git config --local user.email "propagate-bot@pymedphys.com"
           git config --local user.name "PyMedPhys Propagate Bot"
           git add poetry.lock setup.py
-          git commit -m "dummy commit"
+          git commit -m "commit changes to poetry.lock and setup.py"
 
           git status
           git diff-index --quiet HEAD --
+          
+      - name: Fix any issues
```

Comment:
Nice addition!

Reply:
Lemme know when it's working and I can review the job :)

Reply:
Good to go 🙂